template-vue-ts/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "vite-vue-typescript-starter",
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build"
+  },
+  "dependencies": {
+    "vue": "^3.0.0-beta.14"
+  },
+  "devDependencies": {
+    "@vue/compiler-sfc": "^3.0.0-beta.14",
+    "@vuedx/typescript-plugin-vue": "^0.1.0-alpha.12",
+    "typescript": "^3.9.3",
```

Comment:
Just passing through and being nosey. Any reason why this template can't use the latest and greatest version of TS (4.0.3)?